### PR TITLE
Many to many SQL table relations.

### PR DESCRIPTION
**Pull request by simplegoose**
In this pull request, the following was done:

- A table named vets was created.
- Two many-to-many relationship tables were created for specialists and visits.
- Data was inserted into the vets, specialists and visits table.
- The select queries were then written into the queries.sql file.

### Screenshots
Screenshot from 2023-03-31 12-25-17
Screenshot from 2023-03-31 12-26-20

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -26,3 +26,26 @@ UPDATE animals SET owner_id=3 WHERE name='Gabumon' OR name='Pikachu';
...
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (2, 2, '2021-08-07'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (3, 3, '2019-09-29');
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (4, 4, '2020-10-03');
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (4, 4, '2020-11-04'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (5, 2, '2019-01-24'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (5, 2, '2019-05-15'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (5, 2, '2020-02-27'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (5, 2, '2020-08-03'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (6, 3, '2020-05-24'); 
+INSERT INTO VISITS(ANIMAL_ID, VET_ID, DATE_VISITED) VALUES (6, 1, '2021-01-11');
```
Comment (AnselemOdims) [inline data.sql]: - [x] Good job working on this but please kindly include all missing data insert queries.

File: queries.sql [hunk 2]
```diff
@@ -31,4 +31,57 @@ SELECT full_name as owner_full_name, name as animal_name FROM OWNERS
...
   GROUP BY FULL_NAME
-  ORDER BY total_animals DESC;
\ No newline at end of file
+  ORDER BY total_animals DESC;
+
+SELECT vets.name, visits.date_visited FROM VETS
+  JOIN VISITS ON visits.vet_id = vets.id
+  JOIN ANIMALS ON visits.animal_id = animals.id
+  WHERE vets.name = 'William Tatcher'
+  ORDER BY visits.date_visited DESC;
```
Comment (AnselemOdims) [inline queries.sql]: - [x] Good job working on this query but this returns all animals seen by the vet. Please limit it to get the last animal.

File: queries.sql [hunk 3]
```diff
@@ -31,4 +31,57 @@ SELECT full_name as owner_full_name, name as animal_name FROM OWNERS
...
+SELECT animals.name, COUNT(VISITS) as visits FROM ANIMALS
+  JOIN VISITS ON visits.animal_id = animals.id
+  GROUP BY animals.name
+  ORDER BY visits DESC;
+
+SELECT animals.name as animal_name, vets.name as vet_name, visits.date_visited FROM VETS
+  JOIN VISITS ON visits.vet_id = vets.id
+  JOIN ANIMALS ON visits.animal_id = animals.id
+  WHERE vets.name = 'Maisy Smith'
+  ORDER BY visits.date_visited DESC;
```
Comment (AnselemOdims) [inline queries.sql]: - [x] As pointed out above, you should also limit this in order to get the vet's first visit

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @simplegoose , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**